lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -1760,6 +1766,17 @@ ParseResult FIRStmtParser::parseExpImpl(Value &result, const Twine &message,
         builder.create<FIntegerConstantOp>(APSInt(value, /*isUnsigned=*/false));
     break;
   }
+  case FIRToken::lp_path:
+    if (FIRVersion::compare(version, FIRVersion({3, 1, 0})) < 0)
+      return emitError() << "unexpected token: Integers are a FIRRTL 3.1.0+ "
```

Comment:
Probably "path expressions"?
```suggestion
      return emitError() << "unexpected token: path expressions are a FIRRTL 3.1.0+ "
```

Reply:
Good catch! This code conflicts with the improved version checking changes anyway, which would also make this easier to spot.   Anyway, probably not 3.1.0 but tbh I don't know what to do for features not in any particular version yet, I think we just guess where we think it'll land?

Reply:
We could always go in and adjust the version once the feature lands in the FIRRTL spec. And before that just use the current "next" version :wink: